WTG.Analyzers.Test/TestData/CodeContractsAnalyzer/Requires/NonParameter/Source.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Diagnostics.Contracts;
+
+public class Bob
+{
+	public Action Action { get; }
+
+	public void Method1()
+	{
+		Contract.Requires(Action != null);
```

Comment:
Does this have no code-fix?

Reply:
No, no code fix.

This case does not look at any arguments, so we can't really throw an `ArgumentNullException`/`ArgumentException`. Maybe it should be deleted, maybe it should throw an `InvalidOperationException`, `ObjectDisposedException`, ... it's too hard to know what the correct auto-fix is so I figured it would be best to just notify the developer and let them decide manually.